the-badass-todo-list-part-1/input.js

```diff
@@ -18,16 +20,20 @@
         };
         // To prevent user saving multiple times.
         input.disabled = true;
```

Comment:
Well, the closure itself is just the root cause of this problem. But your consideration is in the correct approach.